pyanaconda/ui/lib/subscription.py

```diff
@@ -267,20 +268,34 @@ def register_and_subscribe(payload, progress_callback=None, error_callback=None)
     task_proxy = SUBSCRIPTION.get_proxy(task_path)
     task.sync_run_task(task_proxy)
 
-    # report attaching subscription was successful
+    # check if the current installation source should be overridden by
+    # the CDN source we can now use
+    # - at the moment this is true only for the CDROM source
+    source_proxy = payload.get_source_proxy()
+    if source_proxy.Type in SOURCE_TYPES_OVERRIDEN_BY_CDN:
+        log.debug("subscription thread: overriding current installation source by CDN")
+        switch_source(payload, SOURCE_TYPE_CDN)
+        # disable automatically added treeinfo repos so that
+        # they do not interfere with the CDN
+        for repo in payload.data.repo.dataList():
+            if repo.treeinfo_origin:
+                repo.enabled = False
```

Comment:
Please remove this code. It will be handled by https://github.com/rhinstaller/anaconda/pull/2761.

Reply:
Dropped. :)